lib/manageiq/cross_repo/repository.rb

```diff
@@ -40,6 +42,11 @@ def ensure_clone
         FileUtils.mkdir_p(path.dirname)
         FileUtils.mv(content_dir, path)
       end
+    rescue => e
+      retries += 1
+      raise if retries > 3
+
```

Comment:
Possibly a `sleep 5` would make sense here.  Thoughts?

Reply:
Maybe?  I'm curious what `travis_retry` does, since it's a similar thing and has a _lot_ of real-world usage

Reply:
I think travis_retry does `sleep 1`